README.md

```diff
@@ -60,6 +60,20 @@ source venv/bin/activate
 pip install -e '.[dev]'
 ```
 
+## Development
+
+### bump2version
+
+This repository uses bump2version to manage dependencies. New releases are pushed to PyPi in the CI pipeline when a new version is committed with a version tag and pushed to the repo.
+
+The development flow should use the following process:
+1. New features and bug fixes should be pushed to `dev`
+2. When tests have passed a new development version is ready to be release, use `bump2version major|minor|patch`. This will commit and create a new version tag with the `-dev` suffix.
+3. Additional fixes/features can be added to the current development release by using `bump2version release`.
```

Comment:
Maybe I'm misreading but I couldn't get the relationship b/w `3.` and `4.`.
Is `bump2version release` typo of `bump2version major|minor|patch` in `3.`?

Reply:
It is not a typo. I could probably write this better.

When you got to create a new major/minor/patch version, the version will be changed to something with a `-dev0` suffix.

For example, if your current version is `0.0.1`, then you do `bump2version minor`, the new version will be `0.1.0-dev0`. This is done because you frequently want to make a development release of something for testing reasons prior to making an official release. The build pipeline is designed to build a `0.1.0-dev0` if it gets pushed to the repo and create a PyPi package that can be downloaded with `pip install basicpy==0.1.0-dev0`. The importance of the `dev` suffix is that the development release is not installed when doing something like `pip install basicpy`. You have to actually specify the development version to install it.

Once you make the changes you want and have done testing, you can do `bump2version release`. That would change the version from `0.1.0-dev0` to `0.1.0`.

Reply:
Ooops. It is a typo. It should be `bump2version build`, not `bump2version release`.